locale/ru/LC_MESSAGES/reference/reference_lua/net_box.po

```diff
@@ -1096,120 +1096,3 @@ msgstr ""
 
 msgid "And here starts the example:"
 msgstr "А здесь приведен пример:"
```

Comment:
The translation is missing:

![image](https://user-images.githubusercontent.com/1114511/196370460-76a53128-62e7-4fe9-8f9a-ef31864667db.png)

Reply:
Resolved: updated translations, translated new page